python/IECoreHoudini/TestCase.py

```diff
@@ -37,10 +37,39 @@
 
 ## A class to help implement unit tests for Houdini functionality. It
 # implements setUp() to create a new houdini scene to perform the test in.
+
 class TestCase( unittest.TestCase ) :
 
+	@staticmethod
+	def waitForDebugger( fn ) :
+		"""
+		Decorator for a test or start up function to give
+		the developer time to attach a debugger
+		"""
```

Comment:
Can you add to the description an example of how to use it (for those of us unfamiliar with decorators)?